### PR TITLE
New updates

### PR DESCRIPTION
**Pull request by beckybeauchamp1**
Adam, Joe, and I modified this lesson this morning. Any feedback or input would be much appreciated!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,297 @@
 # http-mvc-intro-rails
+
+## Learning Objectives
+
+- Explain the role of a server-side web application
+  - to build dynamic web pages based on data
```
Comment (beckybeauchamp1) [inline README.md]: We should remove this under the first LO

File: README.md [hunk 2]
```diff
@@ -1 +1,297 @@
...
+
+A server's job is to respond to HTTP requests. In order to talk about how Rails functions, we need to talk about how HTTP requests work.
+
+Every **HTTP request** consists of a request **method** and **path**. The path is the URL to which the request is being sent. That's pretty familiar. However, your browser always sends that request in a particular way that gives the server a hint as to the "point" of the request. This "particular way" is the method.
+
+Once the request is made, the server grabs all sorts of information and begins to build its **response**. It renders the data on a view (HTML), and then sends it back to the client to see it on the browser.
+
+### HTTP Methods (5 min)
+
+HTTP defines five main methods, each of which corresponds to one of the CRUD functionalities.
```
Comment (joe-gz) [inline README.md]: need to either change this to 4, or add PATCH back in and talk about it

File: README.md [hunk 3]
```diff
@@ -1 +1,297 @@
...
+| --- | --- | --- | --- |
+| GET | `/students` | Index | Read information about all students |
+| POST | `/students` | Create | Create a new student |
+| GET | `/students/1` | Show | Read information about the student whose ID is 1 |
+| PUT | `/students/1` | Update | Update the existing student whose ID is 1 with all new content |
+| DELETE | `/students/1` | Delete | Delete the existing student whose ID is 1 |
+
+Note that the path doesn't contain any of the words describing the CRUD functionality that will be executed. That's the method's job.
+
+Let's check out the [ESPN website](http://espn.go.com/mlb/team/_/name/bal). If we go to a specific player's webpage, we can see this same sort of structure in the URL.
```
Comment (joe-gz) [inline README.md]: specific TEAM page - to update..

File: README.md (hunk 2 above)
Comment (beckybeauchamp1) [inline README.md]: We should instead say `HTTP defines the following methods, each of which corresponds to one of the CRUD functionalities.`

File: README.md [hunk 4]
```diff
@@ -1 +1,297 @@
...
+Students should open up the ```app``` folder and explore the code on their own.
+Note anything that reminds you of our in person exercise!
+
+### What does a Rails folder look like? (5 min)
+
+As soon as we generate a Rails app, you can see there are already many folders and files generated from just the one command.
+
+* Note: when we first generated Tunr, we ran the following command:
+```bash
+$ rails new tunr -d postgresql
```
Comment (joe-gz) [inline README.md:177]: Make sure we note to not actually run this code now

File: README.md (hunk 3 above)
Comment (beckybeauchamp1) [inline README.md]: We need to change Action Delete to Destroy for Restful routes

File: README.md [hunk 5]
```diff
@@ -1 +1,297 @@
...
+The `db` folder is one you'll be working in for a bit of time as well. This contains the seed file but additionally it will also contain your migrations which you'll be going over in the next class.
+
+In the main directory there are a couple of files your familiar with, the `Gemfile` and `Gemfile.lock`
+
+### Starting a Rails Server (5 min)
+
+Let's go ahead and look at the final application before we dive into the code.
+>if you want to follow along, clone it down and checkout the solution branch
+
+In my console, I'm run the following command:
```
Comment (joe-gz) [inline README.md]: "I'm going to run the"

File: README.md [hunk 6]
```diff
@@ -1 +1,297 @@
...
+
+>The number `3000` is the port number. This is the default port number in a Rails Application.
+
+Note that this isn't a server anyone else can see, but it's still a server.
+
+(ST-WG) Do you think that the developers of facebook created/updated the facebook application right on https://www.facebook.com? Were all the changes/updates tested on that server? I hope not!
+
+We need a way to develop in our own environment before we just put it on the web. As such, we're going to use our computers as local servers to host our applications until we move it to a production domain. In this way we can test/write code freely in our development environment.
+
+**Let's get into the app!**
```
Comment (joe-gz) [inline README.md]: let's remove this

File: README.md [hunk 7]
```diff
@@ -1 +1,297 @@
...
+
+6. The view takes the objects from the controller and sends a response to the user.
+
+## Break (10 min)
+
+## We Do: In person MVC (30 min)
+
+### Rails Walkthrough (5 min)
+
+Let's walk through a Rails App to get comfortable with it's file structures and identify where we will be configuring the all of the concepts we discussed above! Enter [Tunr](https://github.com/ga-wdi-exercises/tunr_rails_views_controllers/tree/solution)! (yes, use the solution branch!)
```
Comment (beckybeauchamp1) [inline README.md:150]: We need to explicitly list instructions to fork and clone down tunr

**Pull request merged**